src/elexclarity/formatters/const.py

```diff
@@ -12,5 +12,7 @@
     "WV": {
         'PRESIDENT': 'P',
         'U.S. SENATOR': 'S'
+    },
+    "CA": {
```

Comment:
If you wanted clarity IDs to be compatible with our existing race IDs like for GA and WV above, you could define this constant as  
```
"CA": {
        'Recall Gavin Newsom': "I_8808",
        'Office of the Governor Successor to Recall': 'G_recall_choice'
    }
```

Reply:
Yeah, the issue is that the names are different for every county. In Contra Costa the race is called "Shall Gavin Newsom be recalled as Governor?" and in Marin it's just "Recall Question". So I decided I'll just do this on the post processing.